src/main/java/nextstep/security/filter/ExceptionHandlerFilter.java

```diff
@@ -0,0 +1,67 @@
+package nextstep.security.filter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import nextstep.security.exception.AuthenticationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+public class ExceptionHandlerFilter extends OncePerRequestFilter {
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws IOException {
+        try {
+            chain.doFilter(request, response);
+        } catch (Exception e) {
+            createErrorResponse(e, response);
+        }
```

Comment:
```suggestion
        try {
            chain.doFilter(request, response);
        } catch (AuthenticationException e) {
        } catch (IllegalArgumentException e) {
        } catch (Exception e) {
        }
```

instanceof 대신 위와 같이 작성하는게 더 명확할 것 같네요!